openapi/p4pa-io-notification.openapi.yaml

```diff
@@ -4,7 +4,7 @@ info:
   description: Api and Models
   version: 0.0.1
 servers:
-  - url: "http://localhost:8080/notification"
+  - url: "http://localhost:8080/ionotification"
 paths:
   /create/service/{enteId}/{tipoDovutoId}:
```

Comment:
the `/create` is not compliant to RESTful principles. The url should refer to the resource and the http verb should represent the action on the resource. Do you agree?